server/src/repositories/storage.repository.ts

```diff
@@ -214,7 +214,7 @@ export class StorageRepository implements IStorageRepository {
   }
 
   private asGlob(pathToCrawl: string): string {
-    const escapedPath = escapePath(pathToCrawl);
+    const escapedPath = escapePath(pathToCrawl).replaceAll('"', '["]').replaceAll("'", "[']").replaceAll('`', '[`]');
```

Comment:
It's so freaking annoying that there isn't just a library `escapePath` function that does it all...

Reply:
We uncover all kinds of bugs in libraries, I should probably file this as an issue with fast-glob. Likely we are the first ones to even notice